packages/cli/src/utils.js

```diff
@@ -67,3 +68,19 @@ exports.progressBar = total => {
 exports.checkSlashesInUrl = url => {
   return url.replace(/(https?:\/\/)|(\/)+/g, "$1$2")
 }
+
+exports.moveDirectory = (oldPath, newPath) => {
+  const files = fs.readdirSync(oldPath)
+  // check any file exists already
+  for (let file of files) {
+    if (fs.existsSync(file)) {
```

Comment:
This always just checks the CWD, but since `newPath` is a param, we probably want to check instead if `join(newPath, file)` exists here.